qiskit/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -114,9 +114,11 @@ class Grover(AmplitudeAmplifier):
 
     @deprecate_arg(
         "quantum_instance",
-        additional_msg="Instead, use the ``sampler`` argument.",
-        since="0.22.0",
-        pending=True,
+        additional_msg=(
+            "Instead, use the ``sampler`` argument. "
+            "See http://qisk.it/algo_migration for a migration guide."
```

Comment:
Can we use https: links throughout instead of http. For one or two I would do a suggestion - here I think a search replace operation on the files is more in order.